src/components/Spinner/Spinner.js

```diff
@@ -34,6 +34,7 @@ fabric.Spinner = function(target) {
      * @memberOf fabric.Spinner
      */
     function start() {
+        clearInterval(interval);
```

Comment:
There's a stop() method in the code you can call that clears this interval. That would be great if you could swap this clearInterval out and call stop() in its place.